webapp/routes/api.js

```diff
@@ -55,17 +55,34 @@ router.post('/users/register', function (req, res, next) {
 });
 
 router.post('/users/authenticate', function (req, res, next) {
-    User.findOne({email: req.body.email, password: req.body.password}, function (error, user) {
+    User.find({email: req.body.email}, function (error, users) {
```

Comment:
Why was this changed? There should only ever be a single person with an email, right?